node/follower_node_test.go

```diff
@@ -127,3 +146,96 @@ func TestDevModeWarning(t *testing.T) {
 	require.NotNil(t, foundEntry)
 	require.Contains(t, foundEntry.Message, "Follower running on a devMode network. Must submit txns to a different node.")
 }
+
+func TestSyncRoundStartStop(t *testing.T) {
+	// Extend TestSyncRound to the case of starting and stopping the network
+	/*
+		things to take note of:
+		node.Ledger().CommittedRound() is non-deterministic
+		but should be < node.Ledger().Latest()
+
+		don't try to check node.Ledger().CommittedRound() after
+		a Stop()
+
+		It should be comparable to status.SyncRound, but again,
+		this isn't deterministic as the value might have changed
+		between the two evaluations
+	*/
```

Comment:
The inline comments / pr review comments you leave are really helpful. :+1:

Reply:
it's also wrong - I need to update

Reply:
This test is not ready for review... but the e2e test is